AncientReligions/history/characters/irish.txt

```diff
@@ -23509,7 +23509,10 @@
 	culture="irish"
 	father = 7258
 	550.1.1={
-		birth="550.1.1" 
+		birth="550.1.1"
+		effect = {
+			set_global_flag = ancrel
```

Comment:
What do you mean? 
set_global_flag creates the flag, it doesn't need to be defined anywhere else.